components/server/src/authorization/relationship-updater.ts

```diff
@@ -42,6 +44,12 @@ export class RelationshipUpdater {
             },
         });
         if (!fgaEnabled) {
+            if (user.additionalData?.fgaRelationshipsVersion !== undefined) {
+                log.info({ userId: user.id }, `User has been removed from FGA.`);
+                // reset the fgaRelationshipsVersion to undefined, so the migration is triggered again when the feature is enabled
+                AdditionalUserData.set(user, { fgaRelationshipsVersion: undefined });
+                return await this.userDB.storeUser(user);
```

Comment:
I wonder if it make sense to remove the old relations in SpiceDB at this point...
I bet you thought about it, and would love to hear your line of thought. :pray:

Reply:
The relationship-updater takes care of that once the user is enabled again. In the meantime the relationships have no relevance.